site/src/docs/components/login/modules.mdx

```diff
@@ -0,0 +1,474 @@
+
+| Event triggers                               | Required signal props to trigger the listener |
+| -------------------------------------------- | --------------------------------------------- |
+| `apiTokensRemovedTrigger`                    | `payload.type` has a matching event           |
+| `apiTokensRenewalStartedTrigger`             | `payload.type` has a matching event           |
+| `apiTokensUpdatedTrigger`                    | `payload.type` has a matching event           |
+| [Table 19: Api tokens client event triggers] |
+
+Error triggers require the signal to have Api tokens client namespace and type `error`.
+`Signal.payload.type` contains one the <InternalLink href="/components/login/modules#errors-1">errors</InternalLink>.
+
+| Error triggers                               | Required signal props to trigger the listener |
+| -------------------------------------------- | --------------------------------------------- |
+| `invalidApiTokensErrorTrigger`               | `payload.type` has a matching error           |
+| `apiTokensFetchFailedErrorTrigger`           | `payload.type` has a matching error           |
+| `invalidApiTokensUserErrorTrigger`           | `payload.type` has a matching error           |
+| [Table 20: Api tokens client error triggers] |
+
+### Getting signal payloads
+
+Sometimes the significant part of a signal is the payload. There are predefined payload getters for the Api tokens client.
+If the given signal is not from Api tokens client or is not given type, the function returns null. So there is no need to pre-check the signal namespace or type.
+
+| Payload getter                                       | Returns                                     |
+| ---------------------------------------------------- | ------------------------------------------- |
+| `getApiTokensClientEventPayload`                     | null or {type, data} data is usually tokens |
+| `getApiTokensClientErrorPayload`                     | null or ApiTokensError object               |
+| [Table 21: Api tokens client signal payload getters] |
+
+#### Hooks
+
+Api tokens client hooks are listed in the <InternalLink href="/components/login/hooks#api-tokens-client-hooks">hooks section</InternalLink>
+
+### Session poller
+
+The user's session could end outside of the current browser window. The Session poller calls the userinfo endpoint and notifies when it receives an unauthorized or forbidden response.
+Successes and other errors are ignored.
+
+#### Requirements
+
+Polling requires a user and the userManager from the Oidc client. The polling endpoint is fetched from userManager's metadata.
+
+#### Settings
+
+| Property                            | Description                           | Default value |
+| ----------------------------------- | ------------------------------------- | ------------- |
+| `pollIntervalInMs`                  | Optional. Waiting time between polls. | 60000ms       |
```

Comment:
It's 60 0000 ms :)

Reply:
Well yes, I guess programmers don't need to know anything else but it could be easier to convey time to non-programmers in something other than milliseconds 😆 `By default polling happens once a day.` I think would suffice.

Reply:
60 000ms is one minute.

Reply:
Oh.. I don't know what kind of math got me to 16h 40min 🤣 Ok never mind.